nntrainer/layers/rnn.h

```diff
@@ -126,6 +127,8 @@ class RNNLayer : public LayerImpl {
    */
   float dropout_rate;
 
+  float epsilon;
```

Comment:
please add some description for this member variable.

Reply:
added. Thanks.